frankDoc/src/main/java/nl/nn/adapterframework/frankdoc/FrankDocElementSummaryFactory.java

```diff
@@ -0,0 +1,119 @@
+/* 
+Copyright 2021 WeAreFrank! 
+
+Licensed under the Apache License, Version 2.0 (the "License"); 
+you may not use this file except in compliance with the License. 
+You may obtain a copy of the License at 
+
+    http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software 
+distributed under the License is distributed on an "AS IS" BASIS, 
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+
+package nl.nn.adapterframework.frankdoc;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang3.StringUtils;
+
+import nl.nn.adapterframework.frankdoc.model.FrankDocModel;
+import nl.nn.adapterframework.frankdoc.model.FrankElement;
+
+public class FrankDocElementSummaryFactory {
```

Comment:
Kan je in een kort commentaar beschrijven wat deze class doet?

Reply:
Gedaan.